ui/assets/css/_fonts.scss

```diff
@@ -50,3 +50,9 @@
    font-weight: 400;
    src: url('/static/font/Merriweather-Italic.ttf');
  }
+
+ @font-face {
```

Comment:
nitpicking here but can we keep these in order of font-weight to make it easier to scan through if something needs changed?